lib/workers/repository/index.ts

```diff
@@ -29,6 +30,7 @@ export async function renovateRepository(
 ): Promise<ProcessResult> {
   splitInit();
   let config = setAdminConfig(repoConfig);
+  await removeDanglingContainers();
```

Comment:
Shouldn't this only be called once? now it's called for each repo.
```suggestion
```

Reply:
I think it was already called per-repo even if part of global?